src/Common/Libraries/Provider.php

```diff
@@ -28,6 +30,12 @@ public function register() {
 		tribe_register_provider( Installer\Provider::class );
 
 		DB\Config::setHookPrefix( $this->get_hook_prefix() );
+		Assets\Config::set_hook_prefix( $this->get_hook_prefix() );
+		Assets\Config::set_path( Common::instance()->plugin_path . 'src/resources/' );
+		Assets\Config::set_version( Common::VERSION );
+		Assets\Config::set_relative_asset_path( 'src/resources/' );
+		Schema\Config::set_db( DB\DB::class );
```

Comment:
Is this required for this PR about Assets?
I think it will not hurt, since the Schema package is included in the `composer.json` file, but actually not used in Common.

Reply:
Probably not - it's a piece you your original code and I assumed the assets portion was required and the DB one was harmless.